content/docs/quickstart.mdx

```diff
@@ -1,49 +1,76 @@
 ---
 # cSpell:ignore thisisunsafe, genkey, noout
 
-title: Run Pomerium Core With Docker
+title: Secure a Web App With Pomerium Core
 lang: en-US
 sidebar_label: Quickstart
 pagination_prev: null
 pagination_next: null
-description: Get Pomerium up and running quickly with Docker.
+description: In this quickstart guide, you'll learn how to run Pomerium Core in a Docker container and secure access to a web application.
 keywords:
   [
     pomerium,
     identity access proxy,
-    oidc,
-    docker,
     reverse proxy,
     containers,
     identity aware proxy,
-    quickstart pomerium,
+    quickstart,
     docker,
-    docker-compose,
+    docker compose,
+    core,
+    pomerium core,
   ]
 ---
 
 import ConfigDocker from '@site/content/examples/config/config.docker.yaml.md';
 import DockerCompose from '@site/content/examples/docker/basic.docker-compose.yml.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Secure a Web Application With Pomerium
+
+Welcome to Pomerium! This quickstart shows you how to set up Pomerium Core to proxy requests to an internal web application in a containerized Docker environment.
+
+**Steps to complete**:
+
+1. Configure [Pomerium Core](/docs/deploy/core) and the [Verify](https://verify.pomerium.com/) web app to run in Docker containers
+2. Access the Verify web app behind Pomerium
+
+**Time to complete:** 5 minutes
+
+:::note
+
+Check out [**Pomerium Fundamentals**](/docs/courses/fundamentals/get-started) for step-by-step instructions on how to configure and run Pomerium Core to protect your apps and services.
 
-# Run Pomerium Core With Docker
+:::
+
+## Before you start
 
-Run Pomerium Core with Docker containers in **under 5 minutes**.
+This quickstart requires [Docker] and [Docker Compose].
 
-The Core quickstart uses Pomerium's [**Hosted Authenticate Service**](/docs/capabilities/hosted-authenticate-service), but you can also configure a [**self-hosted authenticate service**](/docs/capabilities/self-hosted-authenticate-service) to integrate with Pomerium.
+:::info
 
-## Prerequisites
+This guide uses Docker to bootstrap Pomerium. However, we support other [**architectures and deployment styles**](/docs/deploy), too.
```

Comment:
We already use the term "bootstrap" elsewhere in the docs, where it means something different (particular settings that you need to configure in Core before setting up Enterprise). I'd recommend we avoid using it here.

suggestion:
```suggestion
While this guide uses Docker, Pomerium supports other [**architectures and deployment styles**](/docs/deploy), too.
```